src/commands/appSettings/ISqlDatabaseConnectionWizardContext.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I'm going to say I think they belong where they are because they are all connections related to `appSettings`.  `AzureWebJobsStoragePromptSteps` and `IAzureWebJobsStorageWizardContext` were already in this folder as well, essentially becoming the basis for the pattern I'm following here.

Reply:
<img src="https://media.tenor.com/DP615vqUzeAAAAAM/ace-attorney-phoenix-wright.gif"/>

My rebuttal would be that they were in that folder back when there were only 4 files. It has now grown to like 12 different files in the app settings folder.

I'm okay with them being in the `appSettings` folder still, but maybe make subfolders within that?

```
- azureWebJobsStorage
- eventHubs
- localSettings
- remoteSettings
- sqlDatabase
```

Maybe something like that?

Reply:
I do like the sounds of that structure more